src/relentless/simulate/lammps.py

```diff
@@ -202,6 +203,7 @@ def _call_commands(self, sim):
         masses = mpi.world.bcast(masses)
         sim.masses.update(masses)
         dim_safe = mpi.world.bcast(dim_safe)
+        snap = mpi.world.bcast(snap)
```

Comment:
I don't think this is likely to work, since you are serializing and broadcasting a ton of data. What is it that actually needs to be shared across ranks—just the bond types? We can probably do that by making a dictionary from the label map and broadcasting that.

Reply:
This comment still applies. This will need to be handled differently than broadcasting a snapshot. Let me know if you are unsure what to do here.